### DISCUSSION TITLE
[A-L] (2023/24) Foglio 10 - Esercizio 4

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#329] [Esercizi - Viaggi/Piazza] [answered]
![Screenshot 2023-11-21 105421](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/b144e937-05d1-47d4-834a-85788972fa56)

**Comment by Elia-Belli** [comment]
$q(x)\in Span(p)\iff \alpha\cdot p(x)=q(x)$ per qualche $\alpha \in \mathbb{R}$
Allora è evidente che $q(x)\notin Span(p)$ siccome l'unico modo per far scomparire il termine noto e il termine di 1° grado è che $\alpha =0$ ma i coefficienti di $x^2$ e $x^4$ sono diversi da $0$.

**Comment by daromacarlo** [comment]
![Immagine 2023-11-21 221420](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/24065063-4bfc-4edc-9c27-061224968c8f)

**Comment by Elia-Belli** [comment, marked as answer]
# Soluzione Ufficiale
![Screenshot 2023-11-29 112530](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/70552f50-a4ce-441f-9503-ad0ea7a58656)

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/d9d1df3e-8c29-417a-8ce2-e1c55e0f6da1)
